IdentityCore/src/broker_operation/response/MSIDDeviceInfo.h

```diff
@@ -55,6 +55,12 @@ typedef NS_ENUM(NSInteger, MSIDPreferredAuthMethod)
     MSIDPreferredAuthMethodQRPIN
 };
 
+typedef NS_ENUM(NSInteger, MSIDSsoProviderType)
```

Comment:
Would it be beneficial to make this enum available in a separate header file?

Reply:
There are other enums in this file as of today, we can move out when more enums coming for better management